content/en/docs/reference/_index.md

```diff
@@ -35,6 +35,9 @@ client libraries:
 
 - [Kubernetes Go client library](https://github.com/kubernetes/client-go/)
 - [Kubernetes Python client library](https://github.com/kubernetes-client/python)
+- [Kubernetes Java client library](https://github.com/kubernetes-client/java)
+- [Kubernetes dotnet client library](https://github.com/kubernetes-client/csharp)
+- [Kubernetes JavaScript client library](https://github.com/kubernetes-client/csharp)
```

Comment:
Its link should be https://github.com/kubernetes-client/javascript

Reply:
Fixed